e2e/cypress/integration/signin_authentication/authentication_spec.js

```diff
@@ -0,0 +1,301 @@
+
+            // # Go to Security > Multi-factor Authentication > Edit and Remove MFA
+            cy.get('#securityButton').should('be.visible').click();
+            cy.get('#mfaEdit').should('be.visible').click();
+            cy.findByText('Remove MFA from Account').should('be.visible').click();
+
+            // # Log out
+            cy.apiLogout();
+
+            // # Log in again
+            cy.visit('/login');
+            fillCredentialsForUser(testUser);
+
+            // * Login should be successful without having to enter an MFA code.
+            cy.url().should('include', 'town-square');
+        });
+    });
+
+    it('MM-T406 Sign In Forgot password - Email address not on server (but valid) Focus in login field on login page', () => {
+        // # On a server with site URL and email settings configured (such as rc.test.mattermost.com):
+        // # Go to the login page where you enter username & password
+        cy.visit('/login').wait(timeouts.FIVE_SEC);
+
+        // # Verify focus is in first login field
+        cy.focused().should('have.id', 'loginId');
+
+        // # Click "I forgot my password"
+        cy.findByText('I forgot my password.').should('be.visible').click();
+        cy.url().should('contain', '/reset_password');
+
+        // # Enter an email that doesn't have an account on the server (but that you CAN receive email at)
+        cy.findByPlaceholderText('Email').type('test@test.com');
+
+        cy.findByText('Reset my password').should('be.visible').click();
+
+        // * User redirected to a page with message
+        // "If the account exists, a password reset email will be sent to: [email address]. Please check your inbox."
+        cy.get('#passwordResetEmailSent').should('be.visible').within(() => {
+            cy.get('span').first().should('have.text', 'If the account exists, a password reset email will be sent to:');
+            cy.get('div b').first().should('have.text', 'test@test.com');
+            cy.get('span').last().should('have.text', 'Please check your inbox.');
+        });
+
+        const baseUrl = Cypress.config('baseUrl');
+        const mailUrl = getEmailUrl(baseUrl);
+
+        // * Verify reset email is not sent.
+        cy.task('getRecentEmail', {username: testUser.username, mailUrl}).then((response) => {
+            // Last email should be something else for the test user.
+            expect(response.data.subject).not.equal('[Mattermost] Reset your password');
+        });
+    });
+
+    it('MM-T409 Logging out clears currently logged in user from the store', () => {
+        // # Login as user A and switch to a different team and channel
+        cy.visit('/login');
+        fillCredentialsForUser(testUser);
+
+        cy.visit(`/${testTeam.name}/channels/off-topic`).wait(timeouts.ONE_SEC);
+
+        // # Logout
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+        cy.get('#logout').should('be.visible').click();
```

Comment:
This and the next 2 similar approach failed on me on server licensed with Cloud. It should have "scrollIntoView", like
```javascript
cy.get('#logout').scrollIntoView().should('be.visible').click();
```
or preferably,
```suggestion
        cy.uiOpenMainMenu('Log Out');
```

Reply:
`cy.uiOpenMainMenu('Log Out');` Failed on me 🤷 `cy.get('#logout').scrollIntoView().should('be.visible').click();` does the job thanks :)